docs/source/index.rst

```diff
@@ -23,10 +21,18 @@ This document introduces Olive and provides some examples to get you started.
    getstarted/installation
    getstarted/quickstart_examples
 
+.. toctree::
+   :maxdepth: 1
+   :caption: EXAMPLES
+
+   examples
+
 .. toctree::
    :maxdepth: 1
    :caption: TUTORIALS
 
+   tutorials/design
+   tutorials/options
    tutorials/configure_systems
```

Comment:
List of options should be part of the user guide, not tutorials.

Reply:
Move options back to overview